includes/polling/applications/backupninja.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+// Polls backupninja statistics from script via SNMP
+use LibreNMS\RRD\RrdDefinition;
+
+$name = 'backupninja';
+$app_id = $app['app_id'];
+if (!empty($agent_data['app'][$name])) {
+    $backupninja = $agent_data['app'][$name];
+} else {
+    $options = '-Oqv';
+    $oid     = '.1.3.6.1.4.1.2021.220.3.1.1.11.98.97.99.107.117.112.110.105.110.106.97';
+    $backupninja  = snmp_get($device, $oid, $options);
+}
+
+echo ' backupninja';
+
+list ($last_actions, $last_fatal, $last_error, $last_warning) = explode(" ", $backupninja);
+
+$rrd_name = array('app', $name, $app_id);
+$rrd_def = RrdDefinition::make()
+    ->addDataset('last_actions', 'GAUGE', 0)
+    ->addDataset('last_fatal', 'GAUGE', 0)
+    ->addDataset('last_error', 'GAUGE', 0)
+    ->addDataset('last_warning', 'GAUGE', 0);
+
+$fields = array(
+                'last_actions'   => intval(trim($last_actions)),
+                'last_fatal'     => intval(trim($last_fatal)),
+                'last_error'     => intval(trim($last_error)),
+                'last_warning'   => intval(trim($last_warning)),
+);
+// DEBUG
+// dump($fields);
```

Comment:
Could you remove this, or replace it by d_echo($field); which will only display when poller is run with "-d" option ?

Reply:
please use "d_echo" here. This allow the debug data to be displayed when "-d" option is used in the poller.php.